browser-test/src/admin/admin_publish.test.ts

```diff
@@ -70,6 +70,57 @@ test.describe(
   },
 )
 
+test.describe(
+  'publishing all programs with disabled programs feature flag on',
+  {tag: ['@uses-fixtures']},
+  () => {
+    const disabledProgram = 'Disabled test program'
+    const publicProgram = 'Public test program'
+    const questionName = 'admin-publish-test-address-q'
+    const questionText = 'admin-publish-test-address-q'
+    // CreateNewVersion implicitly updates the question text to be suffixed with " new version".
+    const draftQuestionText = `${questionText} new version`
+
+    test.beforeEach(async ({page, adminPrograms, adminQuestions}) => {
+      await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
+      await loginAsAdmin(page)
+
+      // Create a disabled program
+      await adminPrograms.addDisabledProgram(disabledProgram)
+      // Create a new question referenced by the pubic program
+      await adminQuestions.addAddressQuestion({questionName, questionText})
```

Comment:
suggestion (non-blocking): use `test.step` so you don't have to add the comments explaining what each step is doing.

Reply:
done! thanks for reminding me I will utilize `test.step` more going forward.